src/boot/efi/boot.c

```diff
@@ -2516,9 +2523,17 @@ static void config_load_all_entries(
 
         if (config->auto_firmware && FLAGS_SET(get_os_indications_supported(), EFI_OS_INDICATIONS_BOOT_TO_FW_UI))
                 config_entry_add_call(config,
+                                      LOADER_UNDEFINED,
                                       L"auto-reboot-to-firmware-setup",
                                       L"Reboot Into Firmware Interface",
-                                      reboot_into_firmware);
+                                      reboot_into_firmware_entry);
+
+        /* find if secure boot signing keys exist and autoload them if necessary
+        otherwise creates menu entries so that the user can load them manually
+        if the secure-boot-enrollment variable is set to no (the default), we do not
+        even search for keys on the ESP */
+        if (config->secure_boot_enrollment != ENROLLMENT_OFF)
```

Comment:
Please move the call to after the `config->entry_count == 0` check. We don't want to deal with enrolling keys if there are otherwise no entries we managed to discover. Also, we prefer multi-line comments starting with `*` each line.